index.js

```diff
@@ -119,13 +118,13 @@ const routerSwitch = () => {
     // Create offer UI settings
     case 'offerPage':
       document.querySelector('title').innerText = defaultTitle + ` || Create Offer`;
-      setCreateOfferListener();
+      // setCreateOfferListener();
       break;
 
     // Apply UI settings
     case 'applyPage':
       document.querySelector('title').innerText = defaultTitle + ` || Apply`;
-      postApplication();
+      // postApplication();
```

Comment:
Do we need to keep commented out code, since GH already keep track of changes ?

Reply:
Agreed, let's remove them! Taken care of in: https://github.com/NoroffFEU/agency.noroff.dev/pull/754